### DISCUSSION TITLE
Get the number of completed tasks at the day into yaml

### DISCUSSION THREAD

**Discussion by tiagobrc** [SilentVoid13/Templater#1062] [Help]
I am looking for a method to add in my template the number of tasks that I completed each day in a daily note template Yaml.

So for example it could use the completed date generated by the tasks plugin to know how much was completed on that day. 

I don't know how hard this would be. Does anyone have any ideas?

2023-03-22

---
Completed_Tasks: 9
---

Thanks

**Comment by welpdx** [comment]
I dont use the tasks plugin so can you take some screenshots of the output of the tasks plugin?

Do does all the tasks that is complete reside in one document or what?


Thanks

**Reply by tiagobrc** [reply]
I guess I could have a note with a tasks block that would output all tasks
a task block would output every task in the system.

It looks something like this:

<img width="615" alt="Screenshot 2023-03-23 at 12 27 54 AM" src="https://user-images.githubusercontent.com/12516337/227103197-21ea4e92-425f-422d-8d4c-d673be6522e4.png">

**Reply by welpdx** [reply]
so would it be helpful to have a script that parses for all the ✅ emojis on the note for the date today, have that number as the number of tasks completed?

Would be neat if you copy pasta that over. Btw, what would this taskblock name be? 
What is your workflow? Do you want to run a templater modal on this taskblock each time and generate a file? What is the exact steps/workflow that you are looking for?
